Makefile

```diff
@@ -434,22 +434,24 @@ catalog-push: ## Push a catalog image.
 
 # A valid Git branch from https://github.com/openshift/oadp-operator
 PREVIOUS_CHANNEL ?= oadp-1.3
+PREVIOUS_CHANNEL_GO_VERSION ?= 1.20
 
 .PHONY: catalog-test-upgrade
-catalog-test-upgrade: TEMP:= $(shell mktemp -d)
 catalog-test-upgrade: PREVIOUS_OPERATOR_IMAGE?=ttl.sh/oadp-operator-previous-$(GIT_REV):1h
 catalog-test-upgrade: PREVIOUS_BUNDLE_IMAGE?=ttl.sh/oadp-operator-previous-bundle-$(GIT_REV):1h
 catalog-test-upgrade: THIS_OPERATOR_IMAGE?=ttl.sh/oadp-operator-$(GIT_REV):1h
 catalog-test-upgrade: THIS_BUNDLE_IMAGE?=ttl.sh/oadp-operator-bundle-$(GIT_REV):1h
 catalog-test-upgrade: CATALOG_IMAGE?=ttl.sh/oadp-operator-catalog-$(GIT_REV):1h
 catalog-test-upgrade: opm login-required ## Prepare a catalog image with two channels: PREVIOUS_CHANNEL and from current branch
-	cp -r ./ $(TEMP)/current
-	git clone --depth=1 git@github.com:openshift/oadp-operator.git -b $(PREVIOUS_CHANNEL) $(TEMP)/$(PREVIOUS_CHANNEL)
-	cd $(TEMP)/$(PREVIOUS_CHANNEL) && IMG=$(PREVIOUS_OPERATOR_IMAGE) BUNDLE_IMG=$(PREVIOUS_BUNDLE_IMAGE) make bundle && \
+	# mkdir test-upgrade && cp -r ./ test-upgrade/current
```

Comment:
removed temp to avoid problems, but now I have a can not copy folder into itself problem. Ideas?

Reply:
use rsync, much preferred way, see:
https://github.com/openshift/oadp-operator/pull/1391/files#diff-8de5118a56011d553059dd47da58cea16ecd90598ef82d0d64bcba3ce9cc0423R67

Reply:
going to check out

Reply:
added rsync